### ISSUE TITLE
Set up for gem development doesn't work as described in README

### ISSUE THREAD

**Issue by timdiggins** [appsignal/appsignal-ruby#657]
## Describe the bug

Given I'm trying to start on development on a PR for the first time on this project
I'm following the instructions on the README.
When I run `bundle exec rake install` 
Then I get deprecations warnings but also an error and traceback:

```
[DEPRECATED] `Bundler.with_clean_env` has been deprecated in favor of `Bundler.with_unbundled_env`. If you instead want the environment before bundler was originally loaded, use `Bundler.with_original_env` (called at .../appsignal-ruby/Rakefile:312)
```

```
NameError: uninitialized constant GEMFILES
.../appsignal-ruby/Rakefile:313:in `block (2 levels) in <top (required)>'
.../gems/bundler-2.1.4/lib/bundler.rb:376:in `block in with_clean_env'
.../ruby-2.5.3/gems/bundler-2.1.4/lib/bundler.rb:677:in `with_env'
.../gems/bundler-2.1.4/lib/bundler.rb:376:in `with_clean_env'
.../appsignal-ruby/Rakefile:312:in `block in <top (required)>'
.../gems/rake-11.3.0/exe/rake:27:in `<top (required)>'
.../bin/ruby_executable_hooks:24:in `eval'
.../bin/ruby_executable_hooks:24:in `<main>'
Tasks: TOP => install
(See full trace by running task with --trace)
```

Looking at Rakefile it does appear that GEMFILES is not defined anywhere.

**Comment by tombruijn** [comment]
Hi @timdiggins!

Looks like I broke that a while ago in #498. I've created a PR with a fix in #659.
Thanks for letting us know!

I think we missed this because we usually configure our build environment per gemfile with something like `export BUNDLE_GEMFILE=$PWD/gemfiles/rails-6.0.gemfile` in our shell. And develop against that environment.

**Comment by timdiggins** [comment]
@tombruijn - thanks!
